data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg

```diff
@@ -918,7 +918,7 @@
         name=victory
         [message]
             speaker=Prince Haldric
-            message= _ "We’ve escaped from the orcs before we were trapped by the ice! Now, on to Southbay."
+            message= _ "We’ve escaped from the orcs before getting trapped by the ice! Now, on to Southbay."
```

Comment:
I don't think I like this choice of word. It could be "being" or "becoming" instead, which might be a little better, but I wonder if there's something even better than those…

Reply:
"ending up"?

Reply:
No, I'd vote for being/becoming in this case.

Reply:
"winding up"?

Reply:
Settled on "We’ve escaped from the orcs before the winter ice could trap us!"

Reply:
Much better than the 'up' suggestions.